src/components/projects.js

```diff
@@ -27,7 +27,7 @@ const Projects = () => {
   `)
   const projectsData = projectsQuery.contentJson.projects
   return (
-    <div>
+    <div className="partners-container">
```

Comment:
This is purely to get the background gradient in, might be a better way to do this